packages/widgets/tests/src/title.spec.ts

```diff
@@ -34,6 +36,19 @@ describe('@lumino/widgets', () => {
         title.label = 'baz';
         expect(called).to.equal(true);
       });
+
+      it('should clear signal if it is disposed', () => {
```

Comment:
Minor comment.

I think it would help somebody reading this test code if the name of this test follows the pattern of the preceding test, since the code is patterned after that test:

```
it('should be emitted when the title state changes')
it('should not be emitted if it is disposed')
```